src/lib/utils/skin.ts

```diff
@@ -175,19 +179,29 @@ export function getFadeCalculatorAndSupportedWeapon(
     for (const [fadeType, calculator] of Object.entries(FADE_TYPE_TO_CALCULATOR)) {
         for (const supportedWeapon of calculator.getSupportedWeapons()) {
             if (asset.market_hash_name.includes(`${supportedWeapon} | ${fadeType}`)) {
-                return [calculator, supportedWeapon.toString()];
+                return {
+                    calculator,
+                    weaponName: supportedWeapon.toString(),
+                    className: fadeType.replace(' ', '-').toLowerCase(),
+                };
             }
         }
     }
 }
 
-export function getFadePercentage(asset: rgAsset, itemInfo: ItemInfo): number | undefined {
-    const fadeCalculatorAndSupportedWeapon = getFadeCalculatorAndSupportedWeapon(asset);
+export function getFadePercentage(
+    asset: rgAsset,
+    itemInfo: ItemInfo
+): {percentage: number; fadeType: string} | undefined {
+    const fadeInfo = getFadeParams(asset);
 
-    if (fadeCalculatorAndSupportedWeapon !== undefined) {
-        const [calculator, supportedWeapon] = fadeCalculatorAndSupportedWeapon;
+    if (fadeInfo !== undefined) {
+        const {calculator, weaponName, className} = fadeInfo;
 
-        return calculator.getFadePercentage(supportedWeapon, itemInfo.paintseed).percentage;
+        return {
+            percentage: calculator.getFadePercentage(weaponName, itemInfo.paintseed).percentage,
+            fadeType: className,
```

Comment:
We should avoid remapping this name to `fadeType`, I'd rather call this `className` here as well or define a `fadeType` enum with a function to get the class name on it.